src/pages/gateway/getting-started.md

```diff
@@ -12,7 +12,7 @@ This guide provides you with the basic steps you need to set up your API managem
 Before you can begin using the API Mesh, you must install:
 
 -  [Node.js]
--  [nvm] 16.x.x
+-  [nvm] 16.x.x (Mac/Linux) or [nvm-windows](https://github.com/coreybutler/nvm-windows)(Windows)
```

Comment:
It is better not to specify the version there since you already have the link to nvm ’s gitHub repo and also because the version can change very often
Also,  I believe it is the node version that we support that you have there. Please note that the api-mesh plugin now supports node v18